decide/src/test/java/org/example/DecideTest.java

```diff
@@ -78,4 +78,28 @@ void condition2_EPSILON_equals_PI_angle_is_PI_divided_by_2_FALSE() {
         assertFalse(decide.condition2());
     }
 
+    @Test
+    void condition3_AREA1_equals_1_max_triangle_area_is_2_TRUE() {
+        int numPoints = 4;
+        Point[] points = {new Point(1, 2), new Point(0,2), new Point(0,0), new Point(2, 0)};
```

Comment:
I think there is a problem here, the points will form a triangle of area 1, but the area has to be strictly larger than 1